src/content/en/updates/2016/10/auxclick.md

```diff
@@ -0,0 +1,115 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: A new auxclick event will be fired for non-primary mouse button clicks.
+
+{# wf_published_on: 2016-10-19 #}
+{# wf_updated_on: 2016-10-19 #}
+{# wf_featured_image: /web/updates/images/2016/10/ic_mouse_black.png #}
+{# wf_tags: chrome55,input #}
+
```

Comment:
Recommendation: add a `{# wf_featured_snippet: ... }` - it's used on the /updates/ landing page and provides an opportunity to give people a bit more of a hint about what the article is about. You **can** use HTML in the `wf_featured_snippet`